packages/design-system/src/components/TextField/TextField.stories.jsx

```diff
@@ -43,6 +43,12 @@ export default {
         'HTML `input` [type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#<input>_types) attribute. If you are using `type=number` please use the numeric prop instead.',
       control: { type: 'text' },
     },
+    labelMask: {
+      control: {
+        type: 'radio',
+      },
+      options: ['CURRENCY_MASK', 'PHONE_MASK', 'SSN_MASK', 'ZIP_MASK'],
```

Comment:
In order for Storybook to show something human-readable in this list of options, we have to actually make this field a string and then convert it to the corresponding mask function before rendering the component in the story.